addons/rose/addon/components/rose/form/fieldset/index.hbs

```diff
@@ -0,0 +1,20 @@
+<fieldset ...attributes class='rose-form-fieldset'>
+    <legend 
+        id={{concat 'title-' this.id}}
+        aria-describedby={{concat 'description-' this.id}}
+        class='rose-form-fieldset-title'
+    >
+        {{yield to='title'}}
+    </legend>
+    <div
+        id={{concat 'description-' this.id}}
+        aria-labelledby={{concat 'title-' this.id}}
```

Comment:
I think in this case labeledby isn't unnecessary, since `legend` serves this semantic purpose and should be understood by assistive technology.